rtptransceiver.go

```diff
@@ -106,6 +185,11 @@ func (t *RTPTransceiver) Stop() error {
 }
 
 func (t *RTPTransceiver) setReceiver(r *RTPReceiver) {
+	if r != nil {
+		t.mu.RLock()
+		r.codecs = t.codecs
```

Comment:
Is it possible that these could get out of date? Maybe we should store the `RTPTransceiver` on the `RTPReceiver` and  `RTPSender` as well.

Reply:
You're right, it's better if we store the `RTPTransceiver ` on the `RTPReceiver` and `RTPSender`. Done.